repositories.bzl

```diff
@@ -87,13 +87,22 @@ IO_GRPC_GRPC_JAVA_OVERRIDE_TARGETS = {
 
 def grpc_java_repositories(bzlmod = False):
     """Imports dependencies for grpc-java."""
+    if not native.existing_rule("dev_cel"):
+        http_archive(
+            name = "dev_cel",
+            strip_prefix = "cel-spec-0.15.0",
+            sha256 = "3ee09eb69dbe77722e9dee23dc48dc2cd9f765869fcf5ffb1226587c81791a0b",
+            urls = [
+                "https://github.com/google/cel-spec/archive/refs/tags/v0.15.0.tar.gz",
```

Comment:
This is available as `cel-spec` on bazel-central-registry (BCR). We should add the "not bzlmod and" part of the condition here and add a bazel_dep() to MODULE.bazel. You'll need to use repo_name in the bazel_dep to set its name to dev_cel. (We'll be able to remove that bazel_dep once we swap to xds from BCR).

```patch
diff --git a/MODULE.bazel b/MODULE.bazel
index b0c923c0c..9c45d5862 100644
--- a/MODULE.bazel
+++ b/MODULE.bazel
@@ -46,6 +46,7 @@ IO_GRPC_GRPC_JAVA_ARTIFACTS = [
 # GRPC_DEPS_END
 
 bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "cel-spec", repo_name = "dev_cel", version = "0.15.0")
 bazel_dep(name = "googleapis", repo_name = "com_google_googleapis", version = "0.0.0-20240326-1c8d509c5")
 bazel_dep(name = "grpc", repo_name = "com_github_grpc_grpc", version = "1.56.3.bcr.1")
 bazel_dep(name = "grpc-proto", repo_name = "io_grpc_grpc_proto", version = "0.0.0-20240627-ec30f58")
diff --git a/repositories.bzl b/repositories.bzl
index 7f92f27a3..af3acc8dd 100644
--- a/repositories.bzl
+++ b/repositories.bzl
@@ -87,7 +87,7 @@ IO_GRPC_GRPC_JAVA_OVERRIDE_TARGETS = {
 
 def grpc_java_repositories(bzlmod = False):
     """Imports dependencies for grpc-java."""
-    if not native.existing_rule("dev_cel"):
+    if not bzlmod and not native.existing_rule("dev_cel"):
         http_archive(
             name = "dev_cel",
             strip_prefix = "cel-spec-0.15.0",
```

Reply:
Thanks! Will give it a go.